release.py

```diff
@@ -115,7 +115,7 @@ def _upload_assets(upload_url):
     if os.path.exists('dist'):
         wheelpaths = [os.path.join('dist', ff) for ff in os.listdir('dist')]
     elif os.path.exists('wheelhouse'):
-        wheelpaths = [os.path.join('wheelhouse', ff) for ff in os.listdir('wheelhouse') if 'manylinux' in ff]
+        wheelpaths = [os.path.join('wheelhouse', ff) for ff in os.listdir('wheelhouse')]
```

Comment:
Why is it no longer necessary to check if `manylinux in ff` here - is that no longer in the file/directory names in `wheelhouse`?

Reply:
Ah yeah... so I used to just use `python setup.py bdist_wheel` for Windows which generated builds in `dist` whereas the Linux builds used a manylinux Docker which then generates the files in `wheelhouse`. I've just switched to use [cibuildwheel](https://github.com/pypa/cibuildwheel) which puts all the wheel files in `wheelhouse`. So now all the wheels (from Windows, and Mac OS [new, was not in `pace-python`] as well as Linux) are in `wheelhouse` but the Win/Mac wheels don't have `manylinux` in their names so the previous code ignored them...

Reply:
OK makes sense, yeh sounds like L115-116 are no longer needed...